downstream/modules/platform/ref-example-platform-ext-database-customer-provided.adoc

```diff
@@ -96,4 +96,52 @@ automationhub_pg_sslmode='prefer'
 # postgres_use_ssl=False
 # postgres_ssl_cert=/path/to/pgsql.crt
 # postgres_ssl_key=/path/to/pgsql.key
+
+# Automation EDA Controller Configuration
+#
+
+automationedacontroller_admin_password='<eda-password>'
+
+automationedacontroller_pg_host='data.acme.org'
+automationedacontroller_pg_port=5432
+
+automationedacontroller_pg_database='automationedacontroller'
+automationedacontroller_pg_username='automationedacontroller'
+automationedacontroller_pg_password='<password>'
+
+# The full routable URL used by EDA to connect to a controller host.
+# This URL is required if there is no Automation Controller configured
+# in inventory.
+#
```

Comment:
```suggestion
# Format example: automation_controller_main_url='https://<hostname>'
#
```

Reply:
Adding this example into comments for https://issues.redhat.com/browse/AAP-13313
Inventory file will be to match